types/node.lua

```diff
@@ -90,105 +106,64 @@ function nodetypes.box:absolute ()
    return clone
 end
 
-function nodetypes.box:lineContribution ()
-   -- Regardless of the orientations, "width" is always in the
-   -- writingDirection, and "height" is always in the "pageDirection"
+--- Returns either the width or the height of the box.
+-- Regardless of the orientations, "width" is always in the writingDirection,
+-- and "height" is always in the "pageDirection"
+--
+-- @treturn SILE.types.length  The width or height of the box, depending on the orientation.
+function box:lineContribution ()
```

Comment:
I never understood this despite multiple attempts.
  - "mistfit" is a weird concept
  - When this method is used in `hbox:scaledWidth` the second argument is always self.width.
Franky incomprehensible, one can never know what is the expected code (all the more with such self-targeting calls to `SU.rationWidth`, another very obscure concept everywhere...

Reply:
Misfit seems like a case of bad naming. This is only used for CJK tate, and I assume means an element whose internal contents and hence dimensioning are rotated 90 degrees from the current flow.

Reply:
> This is only used for CJK tate

Hey, that's right. Must relate to  this feature then (screenshot from an Adobe documentation). -- Latin rotated or not in TTB writing.

![image](https://github.com/user-attachments/assets/ce1532a6-e216-42a0-aa2e-2e19acca3347)